lib/UI/components/footer.dart

```diff
@@ -31,6 +27,19 @@ class FooterLayoutMobile extends StatelessWidget {
 }
 
 class FooterLayoutDesktop extends StatelessWidget {
+  @override
```

Comment:
I didn't touch any of this - I think maybe it got caught in the merge... any thoughts?

Reply:
I might have accidentally reverted something?

Reply:
As far as I can tell it looks like something I left. The footer definitely needs some refactoring